Dockerfile.template

```diff
@@ -1,2 +1,4 @@
 LABEL io.openshift.expose-services="3306:tcp" \
     io.openshift.tags="component,%NAME%"
+RUN apt-get update && apt-get install -y gettext-base python &&\
+    cat /opt/app-root/etc/my.cnf > /etc/mysql/my.cnf
```

Comment:
not possible because container user doesn't have permissions to write there.
I thought about passing mysqld config path to another place, but mariadb initializing script (docker_entrypoint.sh) seems to be wired to it's default location